packages/atomic/src/components/commerce/atomic-commerce-load-more-products/e2e/page-object.ts

```diff
@@ -8,13 +8,25 @@ export class LoadMoreProductsPageObject extends BasePageObject<'atomic-commerce-
 
   summary({index, total}: {index?: number; total?: number} = {}) {
     return this.page.getByText(
-      new RegExp(
-        `Showing ${index ?? '\\d'} of ${total ?? '\\d'} result${total === 1 ? '' : 's'}.`
-      )
+      new RegExp(`Showing ${index ?? '\\d+'} of ${total ?? '\\d+'} result`)
```

Comment:
```suggestion
      new RegExp(`Showing ${index ?? '\\d+'} of ${total ?? '\\d+'} results`)
```

Again as I mentioned earlier, this should probably be "products" though